`graphql-parse-resolve-info` : typescript issue

Hi, thanks for Postgraphile!

I'm sorry to post this because it's a typescript issue, and I don't know the exact fix... Though it's most likely a version issue somewhere.

When using `postgraphile@latest` (or 4.0.1) in `typescript@3.1.6` (the latest), `tsc` fails with:
```
node_modules/graphql-parse-resolve-info/node8plus/index.d.ts:2:16 - error TS2665: 
Invalid module name in augmentation.
Module 'graphql/type/definition.js' resolves to an untyped module at 
 '.../node_modules/graphql/type/definition.js', which cannot be augmented.
```

The offending lines seem to be [graphql-parse-resolve-info/src/index.ts:](https://github.com/graphile/graphile-engine/blob/master/packages/graphql-parse-resolve-info/src/index.ts#L24-L29)

```
// Extend GraphQLResolveInfo for old GraphQL versions
declare module "graphql/type/definition.js" {
  interface GraphQLResolveInfo {
    readonly fieldASTs: ReadonlyArray<FieldNode>;
  }
}
```
as when I comment them out the tsc error goes away. The graphql version installed by `npm` is either `0.13.2` in `postgraphile@4.0.1` or `14.0.2` in latest. I'm guessing newer versions of GraphQL don't allow for being extended?

For now I have excluded `node_modules` from my `tsconfig.json` but am just submitting this so you're aware (or to find out if I've misconfigured something). I made a repro [here](https://github.com/amedeedaboville/postgraphile-tsissue).

--- a/packages/graphql-parse-resolve-info/src/index.ts
+++ b/packages/graphql-parse-resolve-info/src/index.ts
@@ -21,13 +21,6 @@ import {
 import { getArgumentValues } from "graphql/execution/values";
 import * as debugFactory from "debug";
 
-// Extend GraphQLResolveInfo for old GraphQL versions
-declare module "graphql/type/definition.js" {
-  interface GraphQLResolveInfo {
-    readonly fieldASTs: ReadonlyArray<FieldNode>;
-  }
-}
-
 type mixed = {} | string | number | boolean | undefined | null;
 
 export interface FieldsByTypeName {
@@ -85,6 +78,7 @@ export function getAliasFromResolveInfo(
   resolveInfo: GraphQLResolveInfo
 ): string {
   const asts: ReadonlyArray<FieldNode> =
+    // @ts-ignore Property 'fieldASTs' does not exist on type 'GraphQLResolveInfo'.
     resolveInfo.fieldNodes || resolveInfo.fieldASTs;
   for (let i = 0, l = asts.length; i < l; i++) {
     const val = asts[i];
@@ -108,6 +102,7 @@ export function parseResolveInfo(
   options: ParseOptions = {}
 ): ResolveTree | FieldsByTypeName | null | void {
   const fieldNodes: ReadonlyArray<FieldNode> =
+    // @ts-ignore Property 'fieldASTs' does not exist on type 'GraphQLResolveInfo'.
     resolveInfo.fieldNodes || resolveInfo.fieldASTs;
 
   const { parentType } = resolveInfo;

